logging/google/cloud/logging/handlers/middleware/request.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from threading import local
+
+
+_thread_locals = local()
+
+
+def get_request():
```

Comment:
@dhermes are you cool with a module-private function called from within the same library? I don't want to expose this to users.